EarthValley80/EarthValley80/Screen/MyNews/Cells/AchieveSummaryLottieCollectionViewCell.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  AchieveSummaryLottieCollectionViewCell.swift
+//  EarthValley80
+//
+//  Created by LeeJiSoo on 2022/11/06.
+//
+
+import UIKit
+
+import Lottie
+
+final class AchieveSummaryLottieCollectionViewCell: UICollectionViewCell {
+
+    // MARK: - property
+
+    override var isSelected: Bool {
+        didSet {
+            if isSelected {
+                lottieView.play()
+            }
+        }
+    }
+
+    private let addBackgroundView: UIImageView = {
+        let addBackgroundView = UIImageView()
+        addBackgroundView.contentMode = .scaleToFill
+        addBackgroundView.clipsToBounds = true
+        addBackgroundView.layer.cornerRadius = 30
+        // TODO: - 로티를 넣어야합니다.
+        addBackgroundView.backgroundColor = .red
+        return addBackgroundView
+    }()
+    private lazy var lottieView: LottieAnimationView = {
+        let animationView = LottieAnimationView(name: lottieName)
+        animationView.play()
+        animationView.contentMode = .scaleAspectFit
+        animationView.loopMode = .repeat(2)
+        return animationView
+    }()
+    // TODO: - lottie뷰 이름을 랜덤으로 돌려줘야 함
+    private lazy var lottieName: String = ""
+
+    // MARK: - init
+
+    override init(frame: CGRect) {
+        super .init(frame: frame)
+        self.setupLayout()
+    }
+
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+
+    // MARK: - func
+
+    private func setupLayout() {
+        self.contentView.addSubview(self.addBackgroundView)
+        self.addBackgroundView.constraint(to: self, insets: UIEdgeInsets.zero)
+
+        self.addBackgroundView.addSubview(self.lottieView)
+        self.lottieView.constraint(to: self.addBackgroundView, insets: UIEdgeInsets.zero)
+    }
+
+    func setLottieImage(_ lottieImageTitle: String) {
```

Comment:
p4;

```suggestion
    func setLottieImage(to imageTitle: String) {
```

함수 전달 인자 부분 네이밍을 이런 식으로 하는 건 어떨까요? 실제로 사용될 때 `setLottieImage(to: 어쩌구)` 이렇게 쓰이다보니깐 함수를 읽을 때 더 부드럽게 읽히지 않을까 싶어요!

Reply:
오 매개변수 레이블이라는 개념을 까마득하게 잊고 있었네요!! 의견 제시 감사합니다! 🫠 [915afa4](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-EarthValley80/pull/71/commits/915afa4427f45d263b6f978bb4ce0fbf6e2cc185)